new_bug.cgi

```diff
@@ -59,6 +62,27 @@ if (lc($cgi->request_method) eq 'post') {
                 comment      => scalar($cgi->param('comment')),
             });
      delete_token($token);
+
+     my $recipients = { changer => $user };
+     my $bug_sent = Bugzilla::BugMail::Send($new_bug->bug_id, $recipients);
+     $bug_sent->{type} = 'created';
+     $bug_sent->{id}   = $new_bug->bug_id;
+     my @all_mail_results = ($bug_sent);
+
+     foreach my $dep (@{$new_bug->dependson || []}, @{$new_bug->blocked || []}) {
+         my $dep_sent = Bugzilla::BugMail::Send($dep, $recipients);
+         $dep_sent->{type} = 'dep';
+         $dep_sent->{id}   = $dep;
+         push(@all_mail_results, $dep_sent);
+     }
+
+     # Sending emails for any referenced bugs.
+     foreach my $ref_bug_id (uniq @{ $new_bug->{see_also_changes} || [] }) {
+         my $ref_sent = Bugzilla::BugMail::Send($ref_bug_id, $recipients);
+         $ref_sent->{id} = $ref_bug_id;
```

Comment:
I suspect it is from https://github.com/bugzilla/bugzilla/blob/master/post_bug.cgi#L215-L241.

@SebastinSanty -- I think we'll still collect $vars->{sentmail} but we need to stash it in a session and have show_bug display. I'll file a related bug for that.